spacy_llm/tests/tasks/test_textcat.py

```diff
@@ -546,3 +553,65 @@ def test_external_template_actually_loads():
 Here is the text: Combine 2 cloves of garlic with soy sauce
 """.strip()
     )
+
+
+INSULTS = [
+    "Gobbledygooks!",
+    "Filibusters!",
+    "Slubberdegullions!",
+    "Vampires!",
+    "Sycophant!",
+    "Kleptomaniacs!",
+    "Egoists!",
+    "Tramps!",
+    "Monopolizers!",
+    "Pockmarks!",
+    "Belemnite!",
+    "Crooks!",
+    "Miserable earthworms!",
+    "Harlequin!",
+    "Parasites!",
+    "Macrocephalic baboon!",
+    "Brutes!",
+    "Pachyrhizus!",
+    "Toads!",
+    "Gyroscope!",
+    "Bougainvillea!",
+    "Bloodsuckers!",
+    "Nincompoop!",
+    "Shipwreckers!",
+]
+
+
+@pytest.mark.parametrize("n_insults", range(len(INSULTS) + 1))
+def test_textcat_scoring(zeroshot_cfg_string, n_insults):
+    @registry.llm_backends("Dummy")
+    def factory():
+        def b(prompts: Iterable[str]) -> Iterable[str]:
+            for _ in prompts:
+                yield "POS"
+
+        return b
+
+    config = Config().from_str(zeroshot_cfg_string)
+    config["components"]["llm"]["backend"] = {"@llm_backends": "Dummy"}
+    config["components"]["llm"]["task"]["labels"] = "Insult"
+    nlp = assemble_from_config(config)
+
+    examples = []
+
+    for i, text in enumerate(INSULTS):
+        predicted = nlp.make_doc(text)
+        reference = predicted.copy()
+
+        reference.cats = {"Insult": 1.0 if i < n_insults else 0.0}
+
+        examples.append(Example(predicted, reference))
+
+    scores = nlp.evaluate(examples)
+
+    pos = n_insults / len(INSULTS)
+    1 - pos
```

Comment:
?

```suggestion
```

Reply:
Ah thanks. My linter removed the variable name because I never used it.